pkg/controller/migplan/migplan_controller.go

```diff
@@ -344,3 +350,37 @@ func (r ReconcileMigPlan) ensureClosed(plan *migapi.MigPlan) error {
 
 	return nil
 }
+
+// Get whether the plan is `suspended`.
+// A plan is considered`suspended` when a migration is running or the final migration has
+// completed successfully. While suspended, reconcile is limited to basic validation
+// and PV discovery and ensuring resources is not performed.
+func (r *ReconcileMigPlan) planSuspended(plan *migapi.MigPlan) error {
+	limited := false
+	migrations, err := plan.ListMigrations(r)
+	if err != nil {
+		log.Trace(err)
+		return err
+	}
+	for _, m := range migrations {
```

Comment:
Limits migrations when:
1. a migration is running
2. a final migration has succeeded